pysat/tests/test_orbits.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 
 import pandas as pds
+from pandas import Timedelta # noqa
```

Comment:
Why is this line in here and why is flake8 ignored?  Why not just use `pds.Timedelta`?

Reply:
Looking through the file, timedelta is only used as `dt.timedelta`.  This could probably be deleted.

Reply:
Curious, why does `__repr__` need this? How can it need something that isn't called? This seems very un-pythonic and I am confused. 🐒

Reply:
It comes from the fun of  ```__repr__``` Everything needs to be just so. All the way at the end of the `__repr__`, Timedelta.
```
In [12]: test = pysat.Instrument('pysat', 'testing', 
    ...:                         clean_level='clean', 
    ...:                         orbit_info={'index': 'mlt', 'period':np.timedelta64(97, 'm')}, 
    ...:                         update_files=True)                                                                                                                                                                         

In [13]: test.orbits.__repr__()                                                                                                                                                                                             
Out[13]: "pysat.Orbits(inst=pysat.Instrument(platform='pysat', name='testing', inst_id='', clean_level='clean', pad=None, orbit_info={'index': 'mlt', 'period': numpy.timedelta64(97,'m')}, inst_module=None, custom=[], **{}), index='mlt', kind='local time', period=Timedelta('0 days 01:37:00'))"

In [14]: test.orbits.orbit_period                                                                                                                                                                                           
Out[14]: Timedelta('0 days 01:37:00')

In [15]: type(test.orbits.orbit_period)                                                                                                                                                                                     
Out[15]: pandas._libs.tslibs.timedeltas.Timedelta

```

```
In [20]: eval(test.orbits.__repr__())                                                                                                                                                                                       
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-20-b5dc860974c5> in <module>
----> 1 eval(test.orbits.__repr__())

<string> in <module>

NameError: name 'Timedelta' is not defined

```

```
In [21]: from pandas import Timedelta                                                                                                                                                                                       

In [22]: eval(test.orbits.__repr__())                                                                                                                                                                                       
Out[22]: pysat.Orbits(inst=pysat.Instrument(platform='pysat', name='testing', inst_id='', clean_level='clean', pad=None, orbit_info={'index': 'mlt', 'period': numpy.timedelta64(97,'m')}, inst_module=None, custom=[], **{}), index='mlt', kind='local time', period=Timedelta('0 days 01:37:00'))
```